### PR TITLE
fix(accessibility) Improve landmarking of the site

### PR DESCRIPTION
**Pull request by tryzniak**
Here is a PR for #2244 .

- adds a navigation landmark
- adds a search landmark
- adds some labels to links

### CODE REVIEW

File: src/components/Navigation/Navigation.jsx [hunk 1]
```diff
@@ -78,7 +80,7 @@ export default class Navigation extends React.Component {
 
         { Links.filter(link => this._isActive(link) && link.children).map(link => (
           <div className="navigation__bottom" key={ link.title }>
-            <Container className="navigation__inner">
+            <nav className="container navigation__inner">
```
Comment (montogeek) [inline src/components/Navigation/Navigation.jsx]: What is the reason of this change?

Comment (tryzniak) [reply src/components/Navigation/Navigation.jsx]: Ah this one. So when using a screenreader there is a way quickly jump to navigations. By using <nav></nav> we add a navigation landmark.

**Comment by montogeek** [comment]
Thanks!
